app/controllers/miq_policy_controller/alerts.rb

```diff
@@ -155,6 +155,19 @@ def alert_field_changed
       end
     end
 
+    if params[:value_mw_greater_than]
```

Comment:
@lucasponce @abonas why is something as basic as `value_greater_than` made to be middleware-specific?